### PR TITLE
Add grilled grapefruit paloma cocktail recipe

### LINKED ISSUE #882: Add a drink! 🍸
Add your drink recipes by forking and submitting a pull request.

Be sure to check out the [Contributing Guidelines ](https://github.com/alfg/opendrinks/blob/master/CONTRIBUTING.md )before submitting a pull request. :)

Happy hacking!

### PR DESCRIPTION
**Pull request by ferdox2**
fix #882 
Adds grilled grapefruit paloma cocktail recipe.

### CODE REVIEW

File: src/recipes/grilled-grapefruit-paloma-cocktail.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "name": "Grilled Grapefruit Paloma Cocktail",
+    "description": "Here's a fired up twist on a traditional Mexican cocktail. You'll grill the grapefruit and lime to caramelize the fruit and give the juice a subtle smoky edge. Stir it up with tequila or mezcal and top with club soda for a refreshing drink that's perfect all summer long.",
+    "github": "ferdox2",
+    "ingredients": [
+        {
+            "quantity": "2",
+            "ingredient": "Limes"
+        },
```
Comment (alfg) [inline src/recipes/grilled-grapefruit-paloma-cocktail.json:9]:
```suggestion
        {
            "quantity": "2",
            "measure": "",
            "ingredient": "Limes"
        },
```

File: src/recipes/grilled-grapefruit-paloma-cocktail.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        {
+            "quantity": "1/4",
+            "measure": "Cup",
+            "ingredient": "White sugar"
+        },
+        {
+            "quantity": "1",
+            "ingredient": "Red grapefruit, halved widthwise"
+        },
```
Comment (alfg) [inline src/recipes/grilled-grapefruit-paloma-cocktail.json:18]:
```suggestion
        {
            "quantity": "1",
            "measure": "",
            "ingredient": "Red grapefruit, halved widthwise"
        },
```

**Review by alfg — CHANGES_REQUESTED** [review]

File: src/recipes/grilled-grapefruit-paloma-cocktail.json (hunk 1 above)
Comment (ferdox2) [reply src/recipes/grilled-grapefruit-paloma-cocktail.json]: done

File: src/recipes/grilled-grapefruit-paloma-cocktail.json (hunk 2 above)
Comment (ferdox2) [reply src/recipes/grilled-grapefruit-paloma-cocktail.json]: done

**Comment by alfg** [comment]
Thanks!

**Pull request merged**